maestrowf/datastructures/specifications/__init__.py

```diff
@@ -0,0 +1,65 @@
+"""Module containing datastructures for parsing specifications."""
+from importlib import import_module
+from inspect import getmodule
+import logging
+
+LOGGER = logging.getLogger(__name__)
+
+
+class SpecificationFactory(object):
+    """A factory class to retrieve different types of Specifications."""
+
+    _classes = {
+        "yaml": {
```

Comment:
Question: This means you're going to support other specifications in the future?